frontend/src/bundles/common/components/button/button.tsx

```diff
@@ -10,6 +10,7 @@ type Properties = {
     size?: 'md' | 'lg';
     isDisabled?: boolean;
     sx?: SystemStyleObject;
```

Comment:
we dont use sx props

Reply:
It will be fixed in https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/178